getting_started/step_by_step/4_first_run.rst

```diff
@@ -17,8 +17,8 @@ walk across the room, etc.) are very easy to use.
 The game start-up itself is also coordinated using an ESC script.
 
 Scripts are plain text files with an ".esc" extension, created in any text
-editor you choose to use. Create a new ESC script which will be our main game
-script and call it "game.esc". Switch to Godot's **Editor** tab and select
+editor you choose to use. To create a new ESC script which will be our main game
```

Comment:
I disagree with this change.

- The suggestion switches from an imperative mode to an infinitive mode, which doesn't fit the rest of the documentation.
- It also removes the naming of the file, which is helpful later on.

While I appreciate the suggestion, I'm not sure this really clarifies or adds anything.

@balloonpopper you're the one heading up the documentation effort these days, so I'll defer to you as to whether the PR is accepted.

Reply:
I just added it above, the info is still below the line, otherwise it's currently not telling you exactly what to do and it has a redundant line.

Reply:
Actually, your edit removes the part about "game.esc", and calling out "game.esc" helps to reinforce what's being asked. I disagree that it doesn't tell you exactly what to do. I'm not sure which part you considered to be redundant...And like I said, it switches modes.

But like I said, @balloonpopper will have a look and review, and if he likes your change, he can weigh in and I'll defer.

Reply:
I agree with @BHSDuncan. This commit removes more words than the contrary, I don't see the improvement in clarity.

Reply:
```
Scripts are plain text files with an “.esc” extension, created in any text editor you choose to use. Create a new ESC script which will be our main game script and call it “game.esc”. Switch to Godot’s Editor tab and select “New Text File” from the “File” menu:
```
So this describes what they are, then tells you they can be made in any text editor, which is a bit unnecessary for the sake of this instruction. Then it tells you to create a new esc script without describing how to do this, followed by the name of the script. Then we finally get told to switch to the editor tab and make a new file, the file we were told about in the very first step. It was difficult as a completely inexperienced Escoria user (yet competent Godot user) to understand what is going on, especially when working through the tutorial successively.

Reply:
I think I see what your concern is now.

So, perhaps a better way to reword it might be as follows since it more closely matches the tone of the rest of the docs:

`Scripts are plain text files with an “.esc” extension, created in any text editor you choose to use. Create a new ESC script which will be our main game script and will be called “game.esc”. To do this, switch to Godot’s Editor tab and select “New Text File” from the “File” menu:`